app/views/search/index.html.erb

```diff
@@ -110,6 +110,8 @@
           <p class="collection-filters">
             <input type="checkbox" id='has-granules' data-bind="checked: query.hasGranules"/>
             <label for="has-granules">Include collections without granules</label>
```

Comment:
How hard would it be to do the inverse of this granules toggle? It would be preferable to have a checked check box and the text say "Only show collection with granules". I like have a checked check box by default as it draws a little more attention to the feature.